index.html

```diff
@@ -1,15 +1,70 @@
     <link rel="stylesheet" href="style/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Courier+Prime:wght@400;700&family=Crete+Round&display=swap"
+        rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Courier+Prime:wght@400;700&family=Crete+Round&family=Poppins&display=swap"
+        rel="stylesheet">
     <title>Portfolio </title>
 </head>
+
 <body>
-    <header></header>
-    <main></main>
-    <footer></footer>
+    <header>
+
+        <div id="header" class="flexColumn">
+            <div id="headerNav" class="headerCont roboto">
+                <a id="logo" href="#header">
+                    <p>VelzckC0D3</p>
+                </a>
+                <nav id="navBar">
+                    <img id="menu" src="style/img/hamburguer.svg" alt="">
+                </nav>
+            </div>
+        </div>
+
+    </header>
+
+    <main class="flexColumn">
+        <section id="main" class="flexColumn">
+            <div id="mainCont">
+                <h1>Hey There. I'm Alejandro</h1>
+                <h2 class="roboto">I'm a Software Developer</h2>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+                <div>
+                    <ul id="mainIcons">
+                        <li> <a href="https://github.com/VelzckC0D3/" target="_blank" rel="noopener"
+                                aria-label="Github"><img class="icon" src="style/icons/github.svg" alt="GitHub"></a>
+                        </li>
+                        <li> <a href="https://www.linkedin.com/in/velzckcode/" target="_blank" rel="noopener"
+                                aria-label="Linkedin"> <img class="icon" src="style/icons/linkedin.svg"
+                                    alt="Linkedin"></a></li>
+                        <li> <a href="https://www.angellist.com/" target="_blank" rel="noopener" aria-label="AngelList">
+                                <img class="icon" src="style/icons/angelList.svg" alt="AngelList"></a></li>
+                        <li> <a href="https://twitter.com/velzck/" target="_blank" rel="noopener" aria-label="Twitter">
+                                <img class="icon" src="style/icons/twitter.svg" alt="Twitter"></a></li>
+                        <li> <a href="https://medium.com/" target="_blank" rel="noopener" aria-label="Medium"> <img
+                                    class="icon" src="style/icons/medium.svg" alt=""></a></li>
```

Comment:
- [ ] All the icons you are using here are broken. I think this is happening because of the file names. All the images inside your folder start with uppercase letters while in the code you are referencing them using lowercase letters. This is how it is currently looking. Kindly use the correct file names to address this issue. 😎️

<img src="https://user-images.githubusercontent.com/17243325/219465496-f65d3866-9e48-4d99-9735-3a8640726ef9.png" width="300">

Reply:
# Also, done! I had to change the folder name in order to see the changes made on the repository 
![image](https://user-images.githubusercontent.com/92229666/219486578-3cbcd4a7-60b2-43e6-822c-fbc9dba84d6b.png)
![image](https://user-images.githubusercontent.com/92229666/219486867-ccac92f1-a963-40f6-9fe6-cd2f4a80a945.png)